docker/nginx/conf.d/default.conf

```diff
@@ -1,12 +1,13 @@
 server {
-    root /srv/api-platform/web;
+    root /srv/app/web;
+    resolver 127.0.0.11;
```

Comment:
It's not a typo: https://docs.docker.com/engine/userguide/networking/#embedded-dns-server

It should work as expected (and 127.0.0.1 shouldn't). Which version of Docker are you using?